test/toTex_test.js

```diff
@@ -0,0 +1,22 @@
+const assert = require('assert');
+
+const parse = require('../docs/bundle.js');
+const toTex = require('../lib/toTex.js');
```

Comment:
hmmm, was the first thing I tried and didn't work. tried it again right now and it worked. Thanks!